docs/vexim-acl-check-rcpt.conf

```diff
@@ -1,3 +1,18 @@
+# spamtrap write sender IP to BL file when receiving messages to special
+# adresses defined in the file 'CONFDIR/spamtrap_receiver'
+deny
```

Comment:
You really want to deny the message? Then it will be removed from the spammers list. I mean you want to receive spam on this spamtrap.